### ISSUE TITLE
Double Click to close a TAB

### ISSUE THREAD

**Issue by nareshtedla** [Microsoft/vscode#52628]
Some of the editors like notepad++ provide Double Click to close a TAB. It would be a great edition to VS Code.

**Comment by chrmarti** [comment]
Where would you double-click? The tab?

**Comment by nareshtedla** [comment]
Yes. Once I double click on the tab, TAB should be closed. When I am working on several tabs, its a pain to point mouse to the close button on TAB and close the TAB. This can be a feature that one can enable if they want to as well.
